gravitee-gateway-core/src/main/java/io/gravitee/gateway/core/logging/LoggableClientRequest.java

```diff
@@ -45,6 +51,19 @@ public LoggableClientRequest(final Request request) {
         log.getClientRequest().setMethod(this.method());
         log.getClientRequest().setUri(this.uri());
         log.getClientRequest().setHeaders(new HttpHeaders(this.headers()));
+
+        if (this.sslSession() != null) {
+            this.request.metrics().setSslProtocol(this.sslSession().getProtocol());
+            this.request.metrics().setSslLocalPrincipal(this.sslSession().getLocalPrincipal() != null ? this.sslSession().getLocalPrincipal().getName() : null);
+            try {
+                this.request.metrics().setSslPeerPrincipal(this.sslSession().getPeerPrincipal() != null ? this.sslSession().getPeerPrincipal().getName() : null);
+            } catch (SSLPeerUnverifiedException peerEx) {
+                this.request.metrics().setSslLocalPrincipal(SSLInfo.UNVERIFIED);
```

Comment:
I'm not sure to understand why are you setting `setSslLocalPrincipal` while you try/catch the `getPeerPrincipal` method call and `setSslLocalPrincipal`is already set in line 57.

Reply:
Good catch. Thanks. Copy-paste issue